app/v3/hateoas/HateoasLinks.scala

```diff
@@ -26,6 +26,30 @@ trait HateoasLinks {
   private def bsasBasUri(appConfig: AppConfig, nino: String) =
     s"/${appConfig.apiGatewayContext}/$nino"
 
+  private def listUri(appConfig: AppConfig, nino: String): String = bsasBasUri(appConfig, nino)
+
+  private def triggerUri(appConfig: AppConfig, nino: String): String =
+    bsasBasUri(appConfig, nino) + "/trigger"
+
+  private def selfEmploymentBsasUri(appConfig: AppConfig, nino: String, bsasId: String): String =
+    bsasBasUri(appConfig, nino) + s"/self-employment/$bsasId"
+
+  private def ukPropertyBsasUri(appConfig: AppConfig, nino: String, bsasId: String): String =
+    bsasBasUri(appConfig, nino) + s"/property/$bsasId"
+
+  private def foreignPropertyBsasUri(appConfig: AppConfig, nino: String, bsasId: String): String =
+    bsasBasUri(appConfig, nino) + s"/foreign-property/$bsasId"
+
+  private def selfEmploymentAdjustmentUri(appConfig: AppConfig, nino: String, calculationId: String): String =
+    bsasBasUri(appConfig, nino) + s"/self-employment/$calculationId/adjust"
+
+  private def propertyAdjustmentUri(appConfig: AppConfig, nino: String, bsasId: String): String =
+    bsasBasUri(appConfig, nino) + s"/property/$bsasId/adjust"
+
+  private def foreignPropertyAdjustmentUri(appConfig: AppConfig, nino: String, bsasId: String): String =
+    bsasBasUri(appConfig, nino) + s"/foreign-property/$bsasId/adjust"
```

Comment:
the uri defs above are never used. I believe in V2 they are integrated into the Api resources links below instead of the `bsasBasUri` being used